lib/js/components/SurveyQuestions/SurveyQuestionTextarea.vue

```diff
@@ -20,7 +20,7 @@
 
 	border: 1px solid $color-mischka-gray;
 	box-sizing: border-box;
-	box-shadow: inset 0 1px 3px $color-minor-supporting;
+	box-shadow: inset 0 1px 3px var(--raw-gray-400);
```

Comment:
do we need to import old `colors` after this change?

Reply:
Yes, because there is still colors from this file in usage. f.e. `$color-mischka-gray` Of course usages will disappear in the `colors-migration-2.0`.

Reply:
so after we merge both branches, we will end up with not used import. Do you have a plan how to fix that?